src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx

```diff
@@ -0,0 +1,81 @@
+import Link from 'next/link';
+import React, { ReactElement, useState, useEffect } from 'react';
+import { setCookie, getCookie, CookieValueTypes } from 'cookies-next';
+import { Modal } from '../../../../../components/Modal/Modal';
+
+interface CitySelectProps {
+  myValue: string | undefined;
+}
+
+export const CitySelect: React.FC<CitySelectProps> = ({ myValue }): ReactElement => {
+  const [showModal, setShowModal] = useState<boolean>(false);
+  const [selectedValue, setSelectedValue] = useState<string | undefined>(myValue);
```

Comment:
not needed, because the type of `selectedValue` is the same as `myValue`

plus `myValue` is not descriptive, use something else